tensorflow/lite/micro/kernels/reduce_test.cc

```diff
@@ -262,6 +262,29 @@ TF_LITE_MICRO_TEST(MeanInt82DKeepDims) {
       output_zero_point, &params);
 }
 
+TF_LITE_MICRO_TEST(MeanInt162DKeepDims) {
```

Comment:
Am I missing something or is it the case that the test cases are only tesing a single code branch
```cc
      } else if (op_data->input_zp == op_data->output_zp &&
                 op_data->input_scale == op_data->output_scale) {
```

Looks like this is not unique to the int16 port so I'm happy to make a Github issue to come back and fix this for the other types as well.

If you can add in additional test cases for int16 to the current PR, that would be appreciated.